### DISCUSSION TITLE
Importing an existing form with 'replace' option raises an error

### DISCUSSION THREAD

**Discussion by scandella** [solspace/craft-freeform#1634] [Questions & Answers]
### What happened?

Importing n existing form with the replace option raises a database error.

### Errors and Stack Trace (if available)

```shell
SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '2-6' for key 'idx_otorfgfidpwalrqrcifielbnvdkgixkteaod'
```

### How can we reproduce this?

1. Import an existing form with the "replace" option
2. A database error is raised
3. 

### Freeform Edition

Pro

### Freeform Version

5.6.6

### Craft Version

Craft Pro 4.12.7

**Comment by gustavs-gutmanis** [comment]
Hi @scandella

Sorry to hear you're experiencing troubles with the import/export functionality.
Could you please provide a full stack trace of the error? The current submitted error message is too vague to point to the underlying problem.

Thanks.

**Comment by scandella** [comment]
Where can I find the full stack trace? I see nothing relevant in /storage/logs.
The error appears in the Freeform CP. 
![CleanShot 2024-10-21 at 10 38 27](https://github.com/user-attachments/assets/07e2ad41-27cd-42c6-9eeb-533a2b7585f2)

**Comment by gustavs-gutmanis** [comment]
You would need to look in the craft log files, which can be hard to parse.
If you're ok with sending us a log file, could you send it to `support@solspace.com`?

There should be a `web.log` file in the `/storage/logs` directory.
To get only relevant data, you could delete the web.log file, perform another import, and then grab the newly generated file and send it to us or look inside of it and find the SQL error and copy what other info there is for it.
